controller/api/public/client.go

```diff
@@ -252,27 +252,7 @@ func NewExternalClient(controlPlaneNamespace string, kubeAPI *k8s.KubernetesAPI)
 		return nil, err
 	}
 
-	log.Debugf("Starting port forward on [%s]", apiURL)
-
-	wait := make(chan error, 1)
-
-	go func() {
-		if err := portforward.Run(); err != nil {
-			wait <- err
-		}
-
-		portforward.Stop()
```

Comment:
This is probably still needed to make sure `pf.stopCh` receives a message to signal the termination of the connection.

Reply:
If I understand correctly, line 264 was never reached because `portforward.Run()` above blocks until `Stop()` is called.